src/telemetry/telemetryService.ts

```diff
@@ -517,6 +517,7 @@ export default class TelemetryService {
       command,
       error_code: errorCode,
       error_name: errorName,
+      error_message: message,
```

Comment:
We can not track the message itself, because it can contain user's sensitive data.

Reply:
oh sorry this was meant to go in the https://github.com/mongodb-js/vscode/pull/900 PR also... but glad that was caught, had no idea! I'm not too sure what we could to do to improve error tracking beyond this

Reply:
Yeah, this is the cause of the problem in the first place, that we can not track a message, and why Kevin was asking about maybe different specialized tools we could utilize. I'm also currently not sure what we could do to improve error tracking.

Reply:
then I could revert and leave it as is for now; hopefully we'll at least see the reduction in the unknown error we see with exportToPlayground

Reply:
So let's exclude `error_message` for now and think about ways to improve error tracking later in a separate pr.

Reply:
Yep, removed it here